src/renderer/components/table/table-cell.tsx

```diff
@@ -60,24 +86,87 @@ export class TableCell extends React.Component<TableCellProps> {
     const showCheckbox = isChecked !== undefined;
 
     if (checkbox && showCheckbox) {
-      return <Checkbox value={isChecked} />;
+      return <Checkbox value={isChecked}/>;
+    }
+  }
+
+  @observable isResizing = false;
+
+  @computed get columnId(): string {
+    return this.props.id ?? this.props.storageId;
+  }
+
+  @computed get columnSize(): number {
+    return getColumnSize(this.props.tableId, this.columnId) ?? 0;
+  }
+
+  @computed get isResizable(): boolean {
+    return [
+      this.props.isResizable,
+      this.props.tableId,
+      this.columnId,
+    ].every(Boolean);
+  }
+
+  @computed get style(): React.CSSProperties {
+    const styles: React.CSSProperties & Record<string, any> = Object.assign({}, this.props.style);
+
+    if (this.isResizable && this.columnSize) {
+      styles.flexGrow = 0;
+      styles.flexBasis = this.columnSize;
```

Comment:
Maybe reassigning `width` CSS property instead of `flex-basis` would produce more satisfying results?